index.js

```diff
@@ -272,7 +272,9 @@ const D2LHypermediaClasses = {
 		intentList: 'intent-list',
 		organizationIntentList: 'organization-intent-list',
 		outcome: 'outcome',
-		outcomes: 'outcomes'
+		outcomes: 'outcomes',
```

Comment:
`outcomes.outcomes`? 🤷‍♂️

Reply:
That's pre-existing, I just added the comma

Reply:
oh yeah, good point